config.js

```diff
@@ -25,7 +25,7 @@ var config = {};
 
```

Comment:
Please include an entry about the change in this PR in the CHANGES_NEXT_RELEASE file. For instnce:

```
- Fix: default southbound non secure port from 5684 to 5683 (#206)
```

Reply:
Fixed by d884ca4ffdb38394f9a6b9cd64519172222b99ed